imap_processing/ultra/tests/plugins/data_path_fixtures.py

```diff
@@ -0,0 +1,62 @@
+"""Pytest plugin module for test data paths"""
+# Standard
+import sys
+from pathlib import Path
+
+# Installed
+import pytest
+
+from imap_processing import packet_definition_directory
+
+
+@pytest.fixture()
+def test_data_path():
+    """Returns the Path to the test_data directory"""
+    return Path(sys.modules[__name__.split(
+        '.')[0]].__file__).parent / 'ultra'/ 'tests' / 'test_data'
+
+@pytest.fixture()
+def ccsds_path(test_data_path):
+    """Returns the ccsds directory.
+    """
+    return test_data_path / 'L0' / \
+        'Ultra45_EM_SwRI_Cal_Run7_ThetaScan_20220530T225054.CCSDS'
+
+@pytest.fixture()
+def xtce_image_rates_path():
+    """Returns the xtce image rates directory.
+    """
+    return f"{packet_definition_directory}ultra" \
+           f"/P_U45_IMAGE_RATES.xml"
+
+@pytest.fixture()
+def xtce_aux_path():
+    """Returns the xtce auxilliary directory.
+    """
+    return f"{packet_definition_directory}ultra/P_U45_AUXILIARY.xml"
```

Comment:
since we decided to user this path, `imap_processing/{instrument}/packet_definitions/` for packet definition, this will change now, right?